pkg/manager/member/tikv_member_manager_test.go

```diff
@@ -2074,3 +2074,77 @@ func TestGetTiKVConfigMap(t *testing.T) {
 		})
 	}
 }
+
+func TestTransformTiKVConfigMap(t *testing.T) {
+	g := NewGomegaWithT(t)
+	tc := newTidbClusterForTiKV()
+	type testcase struct {
+		name    string
+		version int
```

Comment:
add a field for `v1alpha1.TiKVPessimisticTxn`, the content of this configuration is part of test case

Reply:
updated